app/javascript/components/patient/Patient.js

```diff
@@ -639,11 +639,20 @@ class Patient extends React.Component {
                   </div>
                   <div className="item-group">
                     <div>
-                      <b>Symptom Onset:</b>{' '}
-                      <span>{this.props.details.symptom_onset ? moment(this.props.details.symptom_onset, 'YYYY-MM-DD').format('MM/DD/YYYY') : '--'}</span>
+                      <b>Case Status: </b>
+                      <span>{this.props.details.case_status || '--'}</span>
                     </div>
+                    {this.props.details.first_positive_lab_at && (
+                      <div>
+                        <b>First Positive Lab: </b>
+                        <span>{moment(this.props.details.first_positive_lab_at, 'YYYY-MM-DD').format('MM/DD/YYYY')}</span>
+                      </div>
+                    )}
                     <div>
-                      <b>Case Status:</b> <span>{this.props.details.case_status || '--'}</span>
+                      <b>Symptom Onset: </b>
+                      <span>
+                        {this.props.details.symptom_onset ? moment(this.props.details.symptom_onset, 'YYYY-MM-DD').format('MM/DD/YYYY') : 'No symptom history'}
```

Comment:
```suggestion
                        {this.props.details.symptom_onset ? moment(this.props.details.symptom_onset, 'YYYY-MM-DD').format('MM/DD/YYYY') : 'Asymptomatic'}
```
I'm a little unclear from the mock-ups on what this string is supposed to be but this makes it seem like it should be "Asymptomatic"
![image](https://user-images.githubusercontent.com/8235974/112692155-e11b6c80-8e54-11eb-81c1-8b426f0b5510.png)

Reply:
this language came directly from slide 6 of the mockup

<img width="1152" alt="Screen Shot 2021-03-29 at 5 40 00 PM" src="https://user-images.githubusercontent.com/9956561/112903640-cdbb0c00-90b5-11eb-9675-adcb5d2f5be5.png">

Reply:
The image I attached is from slide 13 of the mock-up, which seems to be different from it shows in slide 6, so that may need to be clarified. It's not clear to me if it should be "No symptom history", "No reported symptoms", or "Asymptomatic"

Reply:
the PO team has confirmed that the language should be 'No symptoms reported' https://github.com/SaraAlert/SaraAlert/pull/747/commits/15a6d7a0c71fee53608f709f7c55fb913842dbd8#diff-3a8f4d7cea8c4c666ac2107b63f4c0931ea09e27bfa420c5d8773bc623e47167R656